templates/zerver/register.html

```diff
@@ -153,22 +153,16 @@
           <div class="help-box">
             <div id="org_type_blob_1" class="blob display-none">
               {% trans %}
-              Create a corporate organization if your users will be members of the
-              same company, where the privacy expectation is that the company's
-              organization administrators may need to have access to your users'
-              private message history. Zulip features that allow organization
-              administrators to take control of accounts or access private message
-              history for other users are only available to corporate organizations.
+              Organization admins will be able to take control of users'
+              accounts and access their private messages. Good for corporate
```

Comment:
I am kind of against using **Good** here. It sounds like we recommend to use this mode for Corporations. I am not sure whether all the corporations need this features.